python/pipelines/pipeline_ops.py

```diff
@@ -160,13 +162,18 @@ def run_pipeline_from_func(
 
 
 def _extract_schema_from_bigquery(
+        project: str,
+        location: str,
         table_name: str,
         table_schema: str,
         ) -> list:
     from google.cloud import bigquery
     from google.api_core import exceptions
     try:
-        client = bigquery.Client()
+        client = bigquery.Client(
+            project=project,
+            #location=location,
```

Comment:
Why do we ignore the location input?

Reply:
This is still to be tested in case feature store is located in another region, different from where the rest of the solution is installed.